packages/components/src/Tag/Tag.stories.tsx

```diff
@@ -29,7 +31,7 @@ const Template: Story<Args> = (args) => <Tag {...args} />;
 
 const defaultArgs = {
   children: "Tag text",
-  color: "warning",
+  color: "warning" as const,
```

Comment:
for learning -- what is `as const` doing?

Reply:
Great question!

The `color` prop is a union of specific string values: https://github.com/chanzuckerberg/edu-design-system/blob/9c66b28dccc71fc4693578909bbbfbc45b3104a0/packages/components/src/Tag/Tag.tsx#L8-L14

and doesn't accept arbitrary strings.

In TS (and Flow) when defining an object like

```ts
const defaultArgs = {
  color: 'warning',
};
```

the type ends up being

```ts
{
  color: string
}
```

By adding the `as const` we're telling TypeScript to instead interpret values as specific values, and it ends up being

```ts
{
  color: "warning"
}
```

which we can pass to the component.

Reply:
Can use `as const` with many types, such as

- `const a = "hi" as const; // => type is "hi", not string`
- `const b = false as const; // => type is false, not boolean`

or objects such as this example.